core/arch/arm/mm/mobj.c

```diff
@@ -155,12 +157,6 @@ struct mobj *mobj_phys_alloc(paddr_t pa, size_t size, uint32_t cattr,
 	enum teecore_memtypes area_type;
 	void *va;
 
-	if ((pa & CORE_MMU_USER_PARAM_MASK) ||
```

Comment:
I tried to find back why this alignment constraint was checked. @jenswi-linaro, do you see any reason why this should be preserved, or maybe moved to other specific places for already implement mobj allocation requests?